problems/my_select.rb

```diff
@@ -1,5 +1,8 @@
 class Array
-  def my_select(&prc)
+  # Define a method `my_select(&prc)` that correctly returns
```

Comment:
Should be `Array#my_select(&prc)` or otherwise specify that it's monkey patched onto the Array class